### ISSUE TITLE
User input improvement

### ISSUE THREAD

**Issue by antonio-fr** [coinsurenz/LibreTX#3]
I suggest that you improve the user interface input of some integers, so that the conversion to long little-endian is performed by the software. It will improve drastically the UX. This can be used for "Input* Index", Hash type, and Version.

For example, the user enters the output index as an integer, and the soft is putting in the right format with pack "<l".
```
import struct
out_idx = 1
out_idx_hex = struct.pack( "<l", out_idx ).encode('hex')
```
1 -> "01000000"

Also, what you call in the interface "Input Index" in the input part at the top, is what is usually called the "Output Index".

**Comment by coinsurenz** [comment]
I did consider this but was split over the educational value of abstracting away the need to manually enter these fields.  There are many wallets that already to this well, my intention here was to strike the right balance between ease of use and still requiring the user to more or less put this thing together themselves.  I will implement this and see how it looks.  Completely agree it would make it a lot less painful to enter the inputs.  

I'd like to keep the input index as it is.  I found similar naming conventions to what you referring to quite confusing when reading documentation for example this [https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki)- the 'prevout' being the input in that case.  I will definitely add clarification to my documentation about this, I've also seen it referred to as the 'VOUT' so there certainly could be some confusion here. 

Thank you for the valuable input,  I'll add these changes shortly
